cmd/server/app/migrate_up.go

```diff
@@ -152,6 +160,43 @@ func ensureAuthzStore(ctx context.Context, cmd *cobra.Command, authzw *authz.Cli
 	return storeID, nil
 }
 
+func migratePermsToFGA(ctx context.Context, store db.Store, authzw *authz.ClientWrapper, cmd *cobra.Command) error {
+	cmd.Println("Migrating permissions to FGA...")
+
+	userlist, err := store.ListUsers(ctx, db.ListUsersParams{
+		Limit:  1000,
+		Offset: 0,
+	})
+	if err != nil {
+		return fmt.Errorf("error while listing users: %w", err)
+	}
+
+	for _, user := range userlist {
+		projs, err := store.GetUserProjects(ctx, user.ID)
+		if err != nil {
+			cmd.Printf("Skipping user %d since getting user projects yielded error: %s\n",
+				user.ID, err)
+			continue
+		}
+
+		for _, proj := range projs {
+			if err := authzw.Write(
+				ctx, user.IdentitySubject, authz.AuthzRoleAdmin, proj.ProjectID,
+			); err != nil && strings.Contains(err.Error(), "already exists") {
+				cmd.Printf("Skipping user %d since it already has role on project %s\n",
+					user.ID, proj.ProjectID)
+				continue
+			} else if err != nil {
+				return fmt.Errorf("error while writing tuple: %w", err)
+			}
```

Comment:
Do we want to skip over the rest of the tuples once we hit a single error, or do we want to try to copy as much as we can?  (I'm really not sure, looking for input here)

Reply:
In particular, this decision is different than the decision on line 177.